content/en/docs/instrumentation/python/_index.md

```diff
@@ -13,19 +13,78 @@ the generation and collection of application telemetry data such as metrics,
 logs, and traces. This documentation is designed to help you understand how to
 get started using OpenTelemetry for Python.
 
+## Version support
+
+OpenTelemetry-Python supports Python 3.6 and higher.
+
 ## Status and Releases
 
 The current status of the major functional components for OpenTelemetry Python
 is as follows:
 
-| Tracing   | Metrics | Logging      |
-| -------   | ------- | ------------ |
-| Stable    | Alpha   | Experimental |
+- **Tracing** ([API][api/t], [SDK][sdk/t]): [Stable][]
+- **Metrics** ([API][api/m], [SDK][sdk/m]): [Alpha][Experimental]
+- **[Logging][]**: [Experimental][]
```

Comment:
Small change here, I would change Tracing -> Traces and Logging -> Logs

Additionally, I would add (SDK) with a link to the logs SDK (https://opentelemetry-python.readthedocs.io/en/stable/sdk/logs.html) for consistency with the other signals listed here

Reply:
Done

Reply:
Thanks @chalin, I've filed https://github.com/open-telemetry/opentelemetry.io/issues/1121 to ensure consistency across all languages

Reply:
Thanks!